modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
 = Upgrade Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: upgrade-cluster.adoc, upgrade-components.adoc
 
-Anypoint Runtime Fabric is a package of components that work together. Updates are delivered to these components in the following methods:
+Anypoint Runtime Fabric is a package of components that work together, and updates are delivered in different ways:
 
-* Cluster-level components are delivered as an updated installer package. These components include Docker, Kubernetes, and the Ops Center.
+* Cluster-level components, which include Docker, Kubernetes, and the Ops Center. These components run on virtual machines or bare-metal servers that act as nodes on the cluster.
```

Comment:
Instead of "cluster-level" components, we could go with "Appliance components" and start leading w/ that terminology

Reply:
Updated this section.